kani-driver/src/args.rs

```diff
@@ -114,6 +114,9 @@ pub struct KaniArgs {
     /// Kani will only compile the crate. No verification will be performed
     #[structopt(long, hidden_short_help(true))]
     pub only_codegen: bool,
+    /// Enable --workspace in cargo kani. Does not effect cargo binary.
```

Comment:
This comment is confusing. It's going to be the documentation shown in `--help`. I suggest stealing wording from `cargo build --help`

Reply:
Suggestion taken with 9d2fd46706f3db5e26eb8debaffe3d26c608277b